files/settings-default.cfg

```diff
@@ -288,6 +288,10 @@ autosave = true
 # Display the time played on each save file in the load menu.
 timeplayed = false
 
+# The maximum number of quick (or auto) save slots to have.
+# If all slots are used, the  oldest save is reused
+max quicksaves = 5
```

Comment:
IMO, a default value should be 1 (a vanilla behaviour).

Reply:
I don't imagine this needs to strictly adhere to vanilla, as a QoL improvement that doesn't affect gameplay.

Reply:
I'm a little on the fence here, but I think I agree with akortunov.

Changing how saves are retained will inevitably change people's saving habits, for better or worse. Right now, you probably have one fairly-recent quicksave, autosave and a large number of normal saves in regular intervals. If more quicksaves are retained as proposed in this, it might be that more people will tend towards the 'lazy' route and almost exclusively use quicksaves with little manual saving. This is a problem because the quick saves will still just be recent, and if I figure out that I messed up my character last weekend then I'll have a lot more trouble trying to find a save that's closest to before the event. I could probably think of a more intelligent save-retaining system that'll discard saves with increasing probability based on how old they are. Whether or not this is practical and transparent enough to the user is another question, though.  

Aside from those specific concerns, I'm thinking of possibly enacting a policy for <i>all</i> new optional features to be off by default at least for a little while, so that any unexpected effects will only affect a subset of people.

Reply:
>  If more quicksaves are retained as proposed in this, it might be that more people will tend towards the 'lazy' route and almost exclusively use quicksaves with little manual saving.

The issue is, players are already "lazy" with regards to saving. It's why multiple quick/autosaves became a thing in the fist place. Especially for an immersive game, you don't want to constantly force players to think about how long its been since their last manual save and if they've done anything since that warrants a new save. It's one thing to get into the habit of pressing F5 occasionally during play, but it's something else to regularly remind yourself to go through the menus to make a save (even worse if you have to come up with a name for the save; for me, I'm terrible at coming up with names/titles for anything and could sit there for several moments trying to think of what to call it... and I'm also pretty bad at remembering to save, partly because of the trouble with naming it).

And these days, it's basically a standard feature for games that don't have permadeath. The lack of it is probably going to do more to disrupt a player's usual play habits more than the other way around. For people that still want an authentic one-quicksave-only experience, the option is still there.

> I'm thinking of possibly enacting a policy for all new optional features to be off by default at least for a little while

I could get behind that, with the consideration that *some* features should change the default after its been given some time under testing.

Reply:
I usually only make 1 save per "session".  When I'm playing, I almost exclusively quicksave because it's easier to remember to hit f5 every few minutes than it is to open a menu and go through the save process.  I only make a "real" save when I'm about to exit the game.  In this case, having multiple quicksaves is almost a necessity, because a bad quicksave can remove several hours of progress (at which point I almost always ragequit).

>If more quicksaves are retained as proposed in this, it might be that more people will tend towards the 'lazy' route

I don't necessarily see anything wrong in this.  I've been playing through Divinity: Original Sin 2 using quicksaves and autosaves almost exclusively.  Sure, if I really screw up in the past, I may not be able go back to that save if it's been deleted, but that's my fault anyway for not doing manual saves.  In fact, I noticed that it changed my manual save process from "end of session save" to "important event I always want the option of revisiting" which has made finding important old saves much easier (since I just name them 1, 2, 3, etc).